### PR TITLE
Added the active regions to the policies

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [x] documentation
- [ ] dependencies

## Description

Added the active regions to the policies.

### CODE REVIEW

File: cloud_governance/policy/policy_operations/aws/zombie_non_cluster/zombie_non_cluster_polices.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def run(self):
                             for policy_dict in policy_result:
                                 policy_dict['region_name'] = self._region
                                 policy_dict['account'] = self._account
-                                self._es_operations.upload_to_elasticsearch(data=policy_dict, index=self._es_index)
+                                self._es_operations.upload_to_elasticsearch(data=policy_dict.copy(), index=self._es_index)
```
Comment (ebattat) [inline cloud_governance/policy/policy_operations/aws/zombie_non_cluster/zombie_non_cluster_polices.py:36]: What the purpose of copy ?

Comment (athiruma) [reply cloud_governance/policy/policy_operations/aws/zombie_non_cluster/zombie_non_cluster_polices.py:36]: I do not want to pass ref value, I need to send the values directly.

Comment (athiruma) [reply cloud_governance/policy/policy_operations/aws/zombie_non_cluster/zombie_non_cluster_polices.py:36]: yes

Comment (ebattat) [reply cloud_governance/policy/policy_operations/aws/zombie_non_cluster/zombie_non_cluster_polices.py:36]: Are u sure its documentation issue ?

Comment (athiruma) [reply cloud_governance/policy/policy_operations/aws/zombie_non_cluster/zombie_non_cluster_polices.py:36]: Added bug label too

Comment (ebattat) [reply cloud_governance/policy/policy_operations/aws/zombie_non_cluster/zombie_non_cluster_polices.py:36]: OK

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**